docs/topics/request-response.rst

```diff
@@ -814,3 +823,7 @@ XmlResponse objects
 
 .. _Twisted Failure: https://twistedmatrix.com/documents/current/api/twisted.python.failure.Failure.html
 .. _bug in lxml: https://bugs.launchpad.net/lxml/+bug/1665241
+.. |twisted.internet.address.IPv4Address| replace:: ``twisted.internet.address.IPv4Address``
```

Comment:
This is awesome, I had no idea this could be done to use both monospace font and a link.

However, since we got [intersphinx](https://www.sphinx-doc.org/en/master/usage/extensions/intersphinx.html) now, you could add an entry to `intersphinx_mapping` (in `docs/conf.py`) for the Twisted API (probably `'twisted': ('https://twistedmatrix.com/documents/current/api', None),`) and use:

```
:class:`twisted.internet.address.IPv4Address`
```

Reply:
Nice, will do.

Reply:
I didn't know it was possible either, I found this approach in a StackOverflow answer :-)
It seems a little bit of a hack to me tough, since it involves two steps to create the link and replace the text. The `intersphinx` approach looks much cleaner, thanks!